client/layout/floating-actions/index.jsx

```diff
@@ -28,7 +29,7 @@ const FloatingActions = ( { isHappychatButtonVisible } ) => (
 			{ isHappychatButtonVisible &&
 				config.isEnabled( 'happychat' ) && <HappychatButton allowMobileRedirect /> }
 		</div>
-		{ /* TODO: Move environment badge here */ }
+		<EnvironmentBadge />
```

Comment:
Now I see why there's a separate div for the vertical items!

Reply:
:P Yeah sorry I didn't get around to explaining that yet! I planned on opening this then linking it through!
I did have separate wrapper components originally, handling verticals and horizontals but it turns out that only the vertical group was needed :)
If we want to position something above `<EnvironmentBadge />`, it should be as simple as wrapping the two in that class 👌